dockerfiles/che/Dockerfile

```diff
@@ -38,3 +38,5 @@ COPY entrypoint.sh /entrypoint.sh
 COPY open-jdk-source-file-location /open-jdk-source-file-location
 ENTRYPOINT ["/entrypoint.sh"]
 ADD eclipse-che.tar.gz /home/user/
+RUN mkdir /logs /data
```

Comment:
I would move these RUN instructions before the ADD eclipse-che.tar.gz instruction as it's mainly the only layer that is changing overt time

Reply:
and merge the two RUN instructions into only one (merge it to the only one RUN instruction)